app/views/partials/chargeSummary/ChargeSummaryNotCodingOutAndNotPayeSelfAssessment.scala.html

```diff
@@ -29,6 +29,7 @@
 @(
     chargeItem: models.financialDetails.ChargeItem,
     latePaymentInterestCharge: Boolean,
+    otherInterestCharge: Boolean,
```

Comment:
I guess there is no need for this flag as its always used in combination with latePaymentInterestCharge ?
so this can be evaluated on the controller level ?